package.json

```diff
@@ -87,8 +87,10 @@
     "parse-script-tags": "^0.1.6",
     "pretty-fast": "^0.2.3",
     "prop-types": "^15.6.0",
-    "react": "16.2.0",
-    "react-dom": "16.2.0",
+    "react": "16.4.1",
+    "react-aria-components":
+      "https://github.com/juanca/react-aria-components.git",
+    "react-dom": "16.4.1",
```

Comment:
react-aria-components needs at least React 16.3, because it uses the new `React.createRef` api to manage focusing.